### ISSUE TITLE
validation dataset performance

### ISSUE THREAD

**Issue by Azai-yx** [HannesStark/dirichlet-flow-matching#5]
Hi, Hannes

Recently, I have been using the pre-trained embeddings of promoter sequences as generation conditions based on your model for training. 

Therefore, due to the slight differences in the models, I am currently unsure about the approximate performance of your model on the validation set in terms of val_recovery, val_loss, and val_perplexity. 

Could you share some information with me? My results list below
![image](https://github.com/user-attachments/assets/718315ec-c97b-4554-a92f-4ad008b292b8)
![image](https://github.com/user-attachments/assets/c2b34bb8-1b34-49e9-bac8-05bcb1673b44)
![image](https://github.com/user-attachments/assets/b2d93c7e-45f3-4dc1-826e-d03eb17b3c80)
